wiki/Ranking_criteria/osu!mania/en.md

```diff
@@ -58,7 +58,7 @@ Overall rules and guidelines apply to every kind of osu!mania difficulty. Rhythm
 - **Two notes cannot be placed in a single column at the same timestamp.**
 - **No column can be left empty.** If you want to use fewer keys, change the number of keys in the Song Setup.
 - **There must not be more than 6 notes pressed at any given moment in Insane or lower difficulties.** Using more than 6 notes must also follow a reasonable spread to the next lower difficulty. This rule does not apply to ends of long notes, as they are released, not pressed.
-- **Beatmaps can only use 4 to 10 keys.** Anything else is not supported for the Ranked section.
+- **Beatmaps can only use 4–10, 12, 14, 16, or 18 keys.** Anything else is not supported for the Ranked section.
```

Comment:
```suggestion
- **Beatmaps can only use 4–10, 12, 14, 16, or 18 keys.**
```
seems unnecessary to me, this should be obvious because it's the ranking criteria.